### PR TITLE
Renamed "cpp-ethereum" to "aleth" 

### LINKED ISSUE #5684: Rename "cpp-ethereum" to "aleth" (whole repo)
341 hits:

https://github.com/ethereum/aleth/search?q=cpp-ethereum

cpp-ethereum github links seem to work because of a redirect. Still, the links should be updated.

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #5684

### CODE REVIEW

File: homebrew/INSTALL_RECEIPT.json [hunk 1]
```diff
@@ -1 +1 @@
-{"used_options":["--with-evmjit"],"unused_options":["--with-gui","--without-gpu-mining","--with-debug","--with-vmtrace","--with-paranoia","--successful"],"built_as_bottle":true,"poured_from_bottle":false,"time":1440062311,"HEAD":"4511a6ad3149f514d161dc6c64505f795a0f9cc1","stdlib":"libcxx","compiler":"clang","source":{"path":"/Users/administrator/slave/build-cpp-osx-develop-brew/brew/cpp-ethereum.rb","tap":null,"spec":"devel"}}
+{"used_options":["--with-evmjit"],"unused_options":["--with-gui","--without-gpu-mining","--with-debug","--with-vmtrace","--with-paranoia","--successful"],"built_as_bottle":true,"poured_from_bottle":false,"time":1440062311,"HEAD":"4511a6ad3149f514d161dc6c64505f795a0f9cc1","stdlib":"libcxx","compiler":"clang","source":{"path":"/Users/administrator/slave/build-cpp-osx-develop-brew/brew/aleth.rb","tap":null,"spec":"devel"}}
```
Comment (gumb0) [inline homebrew/INSTALL_RECEIPT.json]: @chfast should we delete this file?

Comment (gumb0) [reply homebrew/INSTALL_RECEIPT.json]: And the whole homebrew directory?

File: scripts/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # For running this container as non-root user, see
```
Comment (gumb0) [inline scripts/Dockerfile]: Looks to be outdated, is this file still useful? @chfast

Comment (gumb0) [reply scripts/Dockerfile]: Other Dockerfiles modified below, too

File: scripts/duplicates.sh [hunk 3]
```diff
@@ -2,27 +2,27 @@
 # -*- coding: iso-8859-15 -*-
 
 #------------------------------------------------------------------------------
-# Python script to analysis cpp-ethereum commits, and filter out duplicates
+# Python script to analysis aleth commits, and filter out duplicates
 #
-# The documentation for cpp-ethereum is hosted at http://cpp-ethereum.org
+# The documentation for aleth is hosted at http://aleth.org
```
Comment (gumb0) [inline scripts/duplicates.sh]:
```suggestion
# The documentation for aleth is hosted at http://cpp-ethereum.org
```

File: scripts/runalltests.sh [hunk 4]
```diff
@@ -3,45 +3,45 @@
 workdir=$(pwd)
 
 #Clean the previous build
-rm -rf cpp-ethereum 2>/dev/null || true
+rm -rf aleth 2>/dev/null || true
```
Comment (gumb0) [inline scripts/runalltests.sh]: This whole file is not useful anymore, let's delete it

File: scripts/tests.bat [hunk 5]
```diff
@@ -1,26 +1,26 @@
 
 REM ---------------------------------------------------------------------------
-REM Batch file for running the unit-tests for cpp-ethereum for Windows.
+REM Batch file for running the unit-tests for aleth for Windows.
 REM
-REM The documentation for cpp-ethereum is hosted at http://cpp-ethereum.org
+REM The documentation for aleth is hosted at http://aleth.org
```
Comment (gumb0) [inline scripts/tests.bat]: Remove this line

File: scripts/tests.sh [hunk 6]
```diff
@@ -1,27 +1,27 @@
 #!/usr/bin/env bash
 
 #------------------------------------------------------------------------------
-# Bash script to execute the cpp-ethereum tests.
+# Bash script to execute the aleth tests.
 #
-# The documentation for cpp-ethereum is hosted at http://cpp-ethereum.org
+# The documentation for aleth is hosted at http://aleth.org
```
Comment (gumb0) [inline scripts/tests.sh]: Remove this line

File: scripts/upload-homebrew-formula.sh [hunk 7]
```diff
@@ -6,10 +6,10 @@
 echo ">>> Starting the script to upload .rb file to homebrew ethereum"
 rm -rf homebrew-ethereum
 git clone git@github.com:ethereum/homebrew-ethereum.git
-cp webthree-umbrella/build/cpp-ethereum.rb homebrew-ethereum
+cp webthree-umbrella/build/aleth.rb homebrew-ethereum
```
Comment (gumb0) [inline scripts/upload-homebrew-formula.sh]: Probably we should delete this file

**Comment by Patil2099** [comment]
@gumb0 I made all the changes that were requested. Please review it.
Thanks!

**Comment by chfast** [comment]
Sorry for being late to the party.

I believe we need 3 PR here:
1. Replace the header comments ("This file is part of cpp-ethereum") with the new one (https://github.com/ethereum/aleth/blob/master/aleth-vm/main.cpp#L1-L3) everywhere.
2. Replace cpp-ethereum with Aleth in all other comments (non-functional changes).
3. Replace cpp-ethereum in other places like scripts, etc. These would need to be more carefully reviewed because they can break things.

**Comment by gumb0** [comment]
I agree, let's better split this, it's too huge and difficult to review.

**Pull request closed without merging**